app/src/main/java/org/simple/clinic/widgets/ageanddateofbirth/UserInputAgeValidator.kt

```diff
@@ -14,21 +15,26 @@ class UserInputAgeValidator @Inject constructor(
 ) {
   sealed class Result {
     object Valid : Result()
-    object Invalid : Result()
+    sealed class Invalid : Result() {
+      object ExceedsMaxAge : Invalid()
+      object ExceedsMinAge : Invalid()
```

Comment:
Can you rename to `ExceedsMaxAgeLimit` and `ExceedsMinAgeLimit`?

Reply:
Done